libs/documentation/src/lib/components/slide-out/demos/basic/slide-out-template.scss

```diff
@@ -0,0 +1,27 @@
+.container {
```

Comment:
Is this css still needed now or can it be removed?

Reply:
It is needed, this file is specific to what is injected inside the slideout and can vary based on what look and feel is being injected